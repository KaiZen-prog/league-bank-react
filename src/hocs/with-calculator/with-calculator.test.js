import React from "react";
import renderer from "react-test-renderer";
import PropTypes from 'prop-types';
import withCalculator from "./with-calculator";

const MockComponent = (props) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const MockComponentWrapped = withCalculator(MockComponent);

it(`withCalculator is rendered correctly`, () => {
  const tree = renderer.create(
      <MockComponentWrapped/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

MockComponent.propTypes = {
  children: PropTypes.node,
};
