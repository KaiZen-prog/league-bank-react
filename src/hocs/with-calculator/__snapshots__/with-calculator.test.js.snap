// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withCalculator is rendered correctly 1`] = `<div />`;
